lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -169,6 +169,7 @@ export default TestModule.extend({
       Ember.run(function() {
         Ember.setProperties(context, hash);
       });
+      return hash;
```

Comment:
Why doesn't this follow the same pattern as above (returning whatever Ember returns from `setProperties`)?